src/parsers/manifest/dash/indexes/timeline.ts

```diff
@@ -420,7 +420,6 @@ export default class TimelineRepresentationIndex implements IRepresentationIndex
     // an explicit discontinuity with the next one
     if (rangeTo !== nextTimelineItem.start &&
         scaledTime >= rangeUp &&
-        scaledTime <= rangeTo &&
```

Comment:
We might want to update the comment of this function to better explain what `range` means